### ISSUE TITLE
yarn add --dev fable-splitter fable-compiler + Incorrect peer dependency

### ISSUE THREAD

**Issue by mohan-win** [fable-compiler/fable-react-native#57]
I'm getting following warnings when I ran  `yarn add --dev fable-splitter fable-compiler` in my new react native project.

```
warning "@react-native-community/eslint-config > @typescript-eslint/eslint-plugin@1.13.0" has incorrect peer dependency "eslint@^5.0.0".
warning "@react-native-community/eslint-config > @typescript-eslint/parser@1.13.0" has incorrect peer dependency "eslint@^5.0.0".
warning "@react-native-community/eslint-config > eslint-plugin-react@7.12.4" has incorrect peer dependency "eslint@^3.0.0 || ^4.0.0 || ^5.0.0".
warning "@react-native-community/eslint-config > eslint-plugin-react-native@3.6.0" has incorrect peer dependency "eslint@^3.17.0 || ^4 || ^5".
warning "@react-native-community/eslint-config > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
```

Wondering how can I resolve this ?

**Comment by MangelMaxime** [comment]
`fable-splitter` and `fable-compiler` don't have dependencies over this package.

Are you sure the problem is coming from Fable packages? Don't you have this warning if you add another package?

**Comment by mohan-win** [comment]
Oh thanks @MangelMaxime. You could be correct, it might be a warning because of other packages..
